### ISSUE TITLE
GrailsLdapAuthoritiesPopulator overrides final method getRolePrefix.()Ljava/lang/String; in grails 3.1.0.RC

### ISSUE THREAD

**Issue by droggo** [grails-plugins/grails-spring-security-ldap#8]
Following error occur for grails-spring-security-ldap-3.0.0.M1 when using Grails 3.1.0.RC during application start up. It happens for new app without any changes.
```
java.lang.VerifyError: class grails.plugin.springsecurity.ldap.userdetails.GrailsLdapAuthoritiesPopulator overrides final method getRolePrefix.()Ljava/lang/String;
...
 at grails.plugin.springsecurity.ldap.SpringSecurityLdapGrailsPlugin$_doWithSpring_closure1.doCall(SpringSecurityLdapGrailsPlugin.groovy:154) ~[spring-security-ldap-3.0.0.M1.jar:na]
```

**Comment by burtbeckwith** [comment]
Fixed in 2.0.1 (https://github.com/grails-plugins/grails-spring-security-ldap/commit/db4f6928b690f297351404d2f0a55ee1818949ed) and 3.0.1 (https://github.com/grails-plugins/grails-spring-security-ldap/commit/9def43ba891e268c4304801fc3104f5d9c6e64d6)

**Comment by mryon** [comment]
This fix broke the cleanrole() method if the ldap.authorities.clean.prefix parameter is set.
